### ISSUE TITLE
Mobile Accessibility Issue

### ISSUE THREAD

**Issue by nith2001** [Husky-Coding-Project/algo-visualizer#46]
For our Graph component, there is a lack of accessibility for the mobile view. Dragging nodes looks very strange and drags the whole background in addition to the node. Also, adding and removing nodes and edges seems virtually impossible. May need to create an "Add/Remove Node" and "Add/Reomve Edge" button for the graph component in addition to the current features of just clicking and adding/removing nodes to improve accessibility.
